sbin/root-mount

```diff
@@ -136,6 +139,15 @@ factory_reset_if_needed()
 		if factory-reset-lvm $LVM_ROOT_SIZE $LVM_RESERVED_MB; then
 			# Successful recovery.
 
+			if test "$CLEAR_EXTERNAL_MEDIA" = "1"; then
+				CLEAR_EXTERNAL_COMMAND=`which factory-reset-external`
```

Comment:
In initrd?  Anyone who's clever enough to have altered $PATH here to inject something didn't need to.  Still my first instinct was full path as well, I just balked on account of the precedent set with factory-reset-lvm.